pom.xml

```diff
@@ -48,7 +48,7 @@
 
 	<properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<java.version>1.7</java.version>
+		<java.version>1.8</java.version>
```

Comment:
1.7 is officially EOL by [Oracle](https://www.oracle.com/java/technologies/javase/7-support-relnotes.html) since 2022 and  EOL by [OpenJDK](https://access.redhat.com/articles/1299013) since 2020.

Also maven-plugin-api from 3.1.0 onward [requires](https://maven.apache.org/developers/compatibility-plan.html) JDK 8

Besides that, it hinders the usage of JUnit 5 which makes writing parameterized tests so much easier.